plugins/generic/pln/templates/status.tpl

```diff
@@ -13,50 +13,43 @@
-				<th>{translate key="plugins.generic.pln.status.packaged"}</th>
-				<th>{translate key="plugins.generic.pln.status.transferred"}</th>
-				<th>{translate key="plugins.generic.pln.status.received"}</th>
-				<th>{translate key="plugins.generic.pln.status.syncing"}</th>
-				<th>{translate key="plugins.generic.pln.status.synced"}</th>
-				<th>{translate key="plugins.generic.pln.status.updated"}</th>
-				<th>{translate key="plugins.generic.pln.status.local_failure"}</th>
-				<th>{translate key="plugins.generic.pln.status.remote_failure"}</th>
+				<th>{translate key="common.id"}</th>
+				<th>{translate key="common.type"}</th>
+				<th>Checked</th>
+				<th>{translate key="plugins.generic.pln.status.local_status"}</th>
+				<th>{translate key="plugins.generic.pln.status.processing_status"}</th>
+				<th>{translate key="plugins.generic.pln.status.lockss_status"}</th>
+				<th>{translate key="plugins.generic.pln.status.complete"}</th>
 				<th></th>
 			</tr>
 			{iterate from=deposits item=deposit}
 			<tr>
 				<td>{$deposit->getId()}</td>
 				<td>{$deposit->getObjectType()}</td>
-				<td>{$deposit->getDepositObjects()|@count}</td>
-				<td>{if $deposit->getPackagedStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getTransferredStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getReceivedStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getSyncingStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getSyncedStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getUpdateStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getLocalFailureStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td>{if $deposit->getRemoteFailureStatus()}{translate key="plugins.generic.pln.status.yes"}{else}{translate key="plugins.generic.pln.status.no"}{/if}</td>
-				<td><input type="submit" name="reset[{$deposit->getId()}]" class="button" value="{translate key="plugins.generic.pln.status.reset"}"/></td>
+				<td>{$deposit->getLastStatusDate()}</td>
+				<td>{translate key=$deposit->getLocalStatus()}</td>
+				<td>{translate key=$deposit->getProcessingStatus()}</td>
+				<td>{translate key=$deposit->getLockssStatus()}</td>
+				<td>{translate key=$deposit->getComplete()}</td>
+				<td><input type="submit" name="reset[{$deposit->getId()}]" class="button" value="{translate key="common.reset"}" onclick="return confirm('{$confirmReset}')" /></td>
```

Comment:
`$confirmReset` needs escaping (e.g. in case an apostrophe appears in the locale key).